app/src/main/java/com/babylon/wallet/android/domain/usecases/transaction/GetTransactionBadgesUseCase.kt

```diff
@@ -9,16 +10,17 @@ class GetTransactionBadgesUseCase @Inject constructor(
 ) {
 
     suspend operator fun invoke(
-        accountProofs: Set<String>
-    ): List<Badge> = stateRepository.getDAppsDetails(
-        definitionAddresses = accountProofs.toList(),
-        isRefreshing = false
-    ).map { dApps ->
-        dApps.map {
+        addresses: Set<ResourceAddress>
+    ): Result<List<Badge>> = stateRepository.getResources(
```

Comment:
what am I missing here and we changed this from getDAppsDetails to getResources?
Although I now the badges are resources

Reply:
It was wrong. That was the problem. I mean it worked, since the address were a string but now that the addresses are validated we need to specify them as resource addresses. Also we used to store in dApp entity resource addresses. So it just needed to retrieve data from resources.

Reply:
I see, thank you